modules/widgets/flickr/widget.php

```diff
@@ -0,0 +1,33 @@
+<!-- Start of Flickr Widget -->
+<table
```

Comment:
Yeah, and it has been quite cringeworthy to write! 😄 

Fact is: that's the wpcom version code.
The point of this porting is so that wpcom sites will look and behave the same after an AT.
And so, what if someone customized the CSS of their Flickr widget and changing the HTML or the class names will screw it up?